tools/osbuild-image-test

```diff
@@ -321,9 +326,9 @@ class TestCases:
 
     @staticmethod
     def _print_stats(tests: List[TestCase]) -> None:
-        distros = defaultdict(int)
-        archs = defaultdict(int)
-        types = defaultdict(int)
+        distros: Dict[str, Any] = defaultdict(int)
+        archs: Dict[str, Any] = defaultdict(int)
+        types: Dict[str, Any] = defaultdict(int)
```

Comment:
Wouldn't the `defaultdict(int)` here indicate that it should be `Dict[str, int]`? :)

Reply:
:D thanks!